docs/docs/10.2-form-input-binding-sugar.md

```diff
@@ -10,7 +10,7 @@ next: test-utils.html
 
 > Note:
 >
-> If you're new to the framework, note that `ReactLink` is not needed for most applications and should be used cautiously.
+> If you're new to the library, note that `ReactLink` is not needed for most applications and should be used cautiously.
```

Comment:
Makes the antecedent a bit more ambiguous (React vs. ReactLink), but I suppose the sentence is true regardless of which antecedent is chosen.  However, we could probably just blow away the qualifier.  Or better yet, just convert it into a deprecation note:

 > `ReactLink` is deprecated as of React v15.  The recommendation is to explicitly set the value and change handler, instead of using `ReactLink`.